src/Configuration/Config.php

```diff
@@ -49,9 +105,27 @@ public function parseConfig(): Collection
         //'permissions' => $this->parseConfigYaml('permissions.yml'),
         //'extensions' => $this->parseConfigYaml('extensions.yml'),
 
+        $this->setTimestamps($general, $taxonomy, $contentTypes);
+
         return $config;
     }
 
+    private function setTimestamps(...$configs): void
+    {
+        $this->timestamps = [];
+
+        foreach ($configs as $config) {
+            foreach ($config->getFilenames() as $file) {
+                $this->timestamps[$file] = filemtime($file);
+            }
+        }
+
+        $envFilename = dirname(dirname(__DIR__)) . '/.env';
```

Comment:
project root dir path must be replaceable

Reply:
Agree completely, but I suggest we do that at once for all of the places where we do this. (and after we've discussed _how_ to do this. :-) )

Reply:
you can just use %kernel.project_dir% here, no?

Reply:
How would I do that?

Reply:
inject it in constructor